Classes/Controller/Backend/Order/OrderController.php

```diff
@@ -65,6 +71,9 @@ public function listAction(int $currentPage = 1): ResponseInterface
     {
         $this->moduleTemplate = $this->moduleTemplateFactory->create($this->request);
 
+        $buttons = $this->getListButtons();
```

Comment:
Do we need an extra variable here?

Reply:
No, not really, just for clarification. I will remove it.